### PR TITLE
829 filter variants for case include acmg v3 + v31 in gene allowlist presets

### PR DESCRIPTION
**Pull request by stolpeo**

### CODE REVIEW

File: varfish/static/js/filter_form.js [hunk 1]
```diff
@@ -1280,6 +1280,179 @@ let presets_genelist = {
...
+      "ATP7B",
+      "BAG3",
+      "BMPR1A",
+      "BRCA1",
+      "BRCA2",
+      "BTD",
+      "CACNA1S",
+      "CASQ2",
+      "COL3A1",
+      "DES",
```
Comment (your-highness) [inline varfish/static/js/filter_form.js:1301]: DES is not part of ACMGv3

File: varfish/static/js/filter_form.js [hunk 2]
```diff
@@ -1280,6 +1280,179 @@ let presets_genelist = {
...
+      "NF2",
+      "OTC",
+      "PALB2",
+      "PCSK9",
+      "PKP2",
+      "PMS2",
+      "PRKAG2",
+      "PTEN",
+      "RB1",
+      "RBM20",
```
Comment (your-highness) [inline varfish/static/js/filter_form.js:1336]: RBM20 is not part of ACMGv3

File: varfish/static/js/filter_form.js [hunk 3]
```diff
@@ -1280,6 +1280,179 @@ let presets_genelist = {
...
+      "SDHC",
+      "SDHD",
+      "SMAD3",
+      "SMAD4",
+      "STK11",
+      "TGFBR1",
+      "TGFBR2",
+      "TMEM127",
+      "TMEM43",
+      "TNNC1",
```
Comment (your-highness) [inline varfish/static/js/filter_form.js:1353]: TNNC1 is not part of ACMGv3

File: varfish/static/js/filter_form.js [hunk 4]
```diff
@@ -1280,6 +1280,179 @@ let presets_genelist = {
...
+      "TRDN",
+      "TSC1",
+      "TSC2",
+      "TTN",
+      "TTR",
+      "VHL",
+      "WT1",
+    ]
+  },
+  "acmg31": {
```
Comment (your-highness) [inline varfish/static/js/filter_form.js:1367]: This actually misses the genes from v3.1 (BAG3, DES, RBM20, TNNC1, and TTR). Also some duplicates are contained here (see below)

File: varfish/static/js/filter_form.js [hunk 5]
```diff
@@ -1280,6 +1280,179 @@ let presets_genelist = {
...
+      "TNNT2",
+      "TP53",
+      "TPM1",
+      "TRDN",
+      "TSC1",
+      "TSC2",
+      "TTN",
+      "VHL",
+      "WT1",
+      "TMEM127",
```
Comment (your-highness) [inline varfish/static/js/filter_form.js:1443]: From here on all are duplicates (12 genes)

**Review by your-highness — CHANGES_REQUESTED** [review]
Please see comments:
- "acmg30" specifies ACMGv3.1 list (including 4 extra genes)
- "acmg31" specifies ACMGv3 including 12 duplicates

**Comment by stolpeo** [comment]
Ups. I mixed up the lists + some copy & paste error. It should be fixed now. Thanks!

**Comment by stolpeo** [comment]
@your-highness please review again

**Review by your-highness — APPROVED** [review]

**Comment by your-highness** [comment]
> @your-highness please review again

Done! Thanks for dev!

**Review by your-highness — APPROVED** [review]

**Comment by your-highness** [comment]
> @your-highness Somehow there is an approval missing for your review that prevents me from merging this (cleanly). Could you examine the github UI whether you can find this somewhere?

A review of user with write access is required (I don't have write permission to the repos)
image

**Comment by stolpeo** [comment]
@your-highness Thanks, good to know. :-)

**Review by holtgrewe — APPROVED** [review]

**Comment by your-highness** [comment]
> @your-highness Thanks, good to know. :-)

Yes! Please give me write access 😛

**Pull request merged**